backend/spec/features/admin/products/prices_spec.rb

```diff
@@ -0,0 +1,19 @@
+# encoding: utf-8
+require 'spec_helper'
+
+describe "Prices", type: :feature do
+  stub_authorization!
+
+  let(:product) { create(:product_with_option_types, price: "1.99", cost_price: "1.00", weight: "2.5", height: "3.0", width: "1.0", depth: "1.5") }
```

Comment:
Could you please add some new lines in here to make it a bit easier to read, and shorten the line length?

Example:
```Ruby
let(:product) do
  create(
    :product_with_option_types,
    price: "1.99",
    cost_price: "1.00",
    weight: "2.5",
    height: "3.0",
    width: "1.0",
    depth: "1.5"
  )
end
```

I would also like to see some of those removed if possible, and only supply the data which is important to make this spec pass.